src/Setup.php

```diff
@@ -85,6 +87,7 @@ public function __construct() {
         add_filter( 'render_block', [ $filter_image_enhanced, 'add_image_attributes' ], 10, 2 );
         add_filter( 'render_block', [ $filter_mediatext_enhanced, 'add_media_text_attributes' ], 10, 2 );
         add_filter( 'render_block', [ $filter_sitelogo_enhanced, 'add_site_logo_attributes' ], 10, 2 );
+        add_filter( 'allowed_redirect_hosts', [ $filter_redirect_hosts, 'add_allowed_redirect_hosts' ], 10, 2 );
```

Comment:
If I understand what you're saying, I think I like leaving the $filter_s the way they are since they're just instantiating classes and aren't very informative.

You did make me realize I should probably use spacing to separate this add_filter from the render_block filters though. :)